src/v0/util/index.js

```diff
@@ -1328,12 +1328,24 @@ const generateExclusionList = (mappingConfig) =>
  */
 function extractCustomFields(message, payload, keys, exclusionFields) {
   const mappingKeys = [];
+  // Define reserved words
+  const reservedWords = ['__proto__', 'constructor', 'prototype'];
+
+  const isReservedWord = (key) => reservedWords.includes(key);
+
   if (Array.isArray(keys)) {
     keys.forEach((key) => {
       const messageContext = get(message, key);
       if (messageContext) {
         Object.keys(messageContext).forEach((k) => {
-          if (!exclusionFields.includes(k)) mappingKeys.push(k);
+          if (isReservedWord(k)) {
+            throw new InstrumentationError(
```

Comment:
do we need to throw errors or remove them?

Reply:
I think if we throw error it will enhance visibility to the customer itself. But open for suggestions. Tagging @utsabc as we discussed it before

Reply:
looks like these are not intentional to me so skipping would be better than throwing errors

Reply:
Ok.

Reply:
you can update the destination doc to let customers know

Reply:
@koladilip this is a generic util function, used in various destinations